documentation/asciidoc/accessories/ai-camera/details.adoc

```diff
@@ -49,7 +49,7 @@ Once `libcamera` dequeues the image and inference data buffers from the kernel,
 | Floating point array storing the input tensor.
 
 | `CnnOutputTensorInfo`
-| Network specific parameters describing the output tensors structure:
+| Network specific parameters describing the output tensor's structure:
```

Comment:
I think this and below is meant to be plural.

Reply:
The reason I thought it was meant to be singular, is that `CnnOutputTensor` just above is described as "Floating point array storing the output tensor." and `CnnInputTensor` is described as "Floating point array storing the input tensor." ?

Reply:
That should probably be plural as well.  Input tensor is singular.

Reply:
Ahhh - so a single input tensor, but multiple output tensors?

Reply:
Yup, that's correct.